src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/TopLevelState/TopLevelState.ts

```diff
@@ -488,3 +488,20 @@ export var routingErrorDirective = (adhConfig  : AdhConfig.IService) => {
         }]
     };
 };
+
+
+export var goToCameFromDirective = (adhTopLevelState : Service) => {
+    return {
+        restrict: "E",
+        template: "<div class=\"resource-navigation-breadcrumbs\"><a href=\"\" data-ng-click=\"cancel()\">" +
+            "{{\"TR__CANCEL\" | translate}}</a></div>",
```

Comment:
Having `.resource-navigation-bradcrumbs` in here makes it very hard to reuse this directive. IMHO it should be outside.